src/app/Components/ArtworkGrids/InfiniteScrollArtworksGrid.tsx

```diff
@@ -363,7 +379,13 @@ class InfiniteScrollArtworksGrid extends React.Component<Props & PrivateProps, S
       return null
     }
 
-    return React.isValidElement(HeaderComponent) ? HeaderComponent : <HeaderComponent />
+    return React.isValidElement(HeaderComponent) ? (
+      <View onLayout={this.onHeaderLayout}>{HeaderComponent}</View>
+    ) : (
+      <View onLayout={this.onHeaderLayout}>
+        <HeaderComponent />
+      </View>
+    )
```

Comment:
Can we just check for the type of the header component and change this to:

```suggestion
    return (
     <View onLayout={this.onHeaderLayout}>
        <HeaderComponent />
      </View>
    )
```

Reply:
Ok